src/general/DayNightConfiguration.cs

```diff
@@ -24,6 +24,13 @@ public class DayNightConfiguration : IRegistryType
     [JsonProperty]
     public float DaytimeFraction { get; private set; }
 
+    /// <summary>
+    ///   Fraction of the in-game day which does not has sunlight.
```

Comment:
```suggestion
    ///   Fraction of the in-game day which does not have sunlight.
```
I think that's a typo.